frb_codegen/src/main.rs

```diff
@@ -14,20 +12,21 @@ fn main() -> anyhow::Result<()> {
     #[cfg(feature = "serde")]
     let dump_config = raw_opts.dump.clone();
 
-    let configs = config_parse(raw_opts);
-    debug!("configs={:?}", configs);
+    let (all_configs, all_symbols) = config_parse(raw_opts)?;
+    debug!("configs={:?}", all_configs);
 
+    // generation of rust api for ffi
     #[cfg(feature = "serde")]
     if let Some(dump) = dump_config {
-        return dump::dump_multi(&configs, dump).context("Failed to dump config");
+        return dump::dump_multi(&all_configs, dump).context("Failed to dump config");
     }
```

Comment:
nit: I guess this `if` means dump the config, so the newly added comment "generation of rust api for ffi" may not be super exact